### PR TITLE
Use JAXB for Base64 encoding in aws sdk with Java 9+

### PR DESCRIPTION
**Pull request by raunaqmorarka**
Right now running a query with hive connector which writes results to S3 generates the below warning from aws sdk with Java 9+
```
2020-02-22T02:48:09.263+0530	WARN	s3-transfer-manager-worker-1	com.amazonaws.util.Base64	JAXB is unavailable. Will fallback to SDK implementation which may be less performant.If you are using Java 9+, you will need to include javax.xml.bind:jaxb-api as a dependency.
```

### CODE REVIEW

File: presto-main/etc/log.properties [hunk 1]
```diff
@@ -9,3 +9,4 @@ io.prestosql=INFO
 com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory=WARN
 com.ning.http.client=WARN
 io.prestosql.server.PluginManager=DEBUG
+com.amazonaws=WARN
```
Comment (electrum) [inline presto-main/etc/log.properties]: Why do we want to add this to the development configs? Is this related to adding the JAXB dependency?

Comment (raunaqmorarka) [reply presto-main/etc/log.properties]: Adding this helped to discover the warning from aws sdk, I thought it might be useful to discover similar issues in future. I can remove it if it's not useful to have it here.

Comment (electrum) [reply presto-main/etc/log.properties]: INFO is the default level, so this is decreasing the level to WARN. Do you not see warnings without this?

I was concerned that this PR was causing the SDK to log more messages at INFO level and you were thus trying to hide them.

Comment (raunaqmorarka) [reply presto-main/etc/log.properties]: My bad, I somehow mistakenly thought that the warning wasn't showing up with the default config but I've now confirmed that it does. Reverted this change.

**Comment by electrum** [comment]
Thanks!
